app/views/presentations/_sign_up_link.html.erb

```diff
@@ -0,0 +1,5 @@
+<%=
+  if event&.upcoming?
+    link_to 'Sign up', new_presentation_path(prepopulated_event: event.id)
```

Comment:
Mmmmmm I kind of want there to be some reference to what this is for when it hits the controller.
This allows me to have `params[:prepopulated_event]` in my controller. I know what that is.
Having a reference to `params[:event_id]` in my presentations controller is kind of odd. Why do I have a reference to this? What is this being used for?